.github/workflows/ci_workflows.yml

```diff
@@ -3,7 +3,12 @@ name: CI
 on: [push, pull_request]
 
 jobs:
+  download:
+    uses: zacharyburnett/webbpsf/.github/workflows/cache.yml@ci/cache
```

Comment:
Minor: presumably this should be the `spacetelescope` repo here rather than the `zacharyburnett` one? At least after this gets merged in so that that path exists in the space telescope repo?

Reply:
oh yes, this should be changed back before merging (it's just there for testing)